scripts/upgrade.sh

```diff
@@ -23,8 +23,13 @@ fi
 
 # Backup database
 SQL_BACKUP_PATH="$PEERTUBE_PATH/backup/sql-peertube_prod-$(date +"%Y%m%d-%H%M").bak" 
+DB_USER=$($PEERTUBE_PATH/peertube-latest/node_modules/.bin/js-yaml $PEERTUBE_PATH/config/production.yaml | python -c 'import json,sys;obj=json.load(sys.stdin);print obj["'database'"]["'username'"]';)
```

Comment:
May I suggest:

```
$ cd "$PEERTUBE_PATH/peertube-latest"
$ node -e "console.log(require('js-yaml').safeLoad(fs.readFileSync('$PEERTUBE_PATH/config/production.yaml', 'utf8'))['database']['username'])"
$ ...
$ ...
```

To avoid python calls?

Reply:
Sure. I didn't knew how to do that in NodeJS, that's why I did it in Python.

Reply:
Done.